apps/ledger-live-mobile/e2e/page/manager/manager.page.ts

```diff
@@ -27,7 +33,11 @@ export default class ManagerPage {
   }
 
   async waitForManagerPageToLoad() {
```

Comment:
this method is not in use anywhere, do we need to keep it anyway?

Reply:
Deleted